pymatgen/io/ase.py

```diff
@@ -38,6 +31,13 @@ def __init__(self, *args, **kwargs):
             raise no_ase_err
 
 
```

Comment:
i would stick to single blank line above `if TYPE_CHECKING:`. would be nice if `ruff` enforced this

Reply:
I tried to remove the extra blank line but `ruff` added it back. `ruff` seem to have a weird interaction with `if TYPE_CHECKING:` for some reason?

Reply:
Thanks for reviewing. I guess for such issues we'd better wait until `ruff` has some solution (or perhaps already has but we don't know about), and run `ruff` to fix all issues.